index.js

```diff
@@ -0,0 +1,50 @@
+
+// . reduce ()
+// obj
+// we have ; 
+// arrays, properties, objects(string, numbers)
+
+function totalSalaryTest(lineup) {
```

Comment:
This function is adding up all the salaries so something like `calculateTotalSalary` would be a better name.